app/uk/gov/hmrc/vulnerabilities/persistence/VulnerabilitiesRepository.scala

```diff
@@ -39,18 +43,58 @@ class VulnerabilitiesRepository @Inject()(
     IndexModel(Indexes.ascending("teams"), IndexOptions().name("teams").background(true)),
   ),
 ) {
-  def search(service: Option[String] = None, id: Option[String] = None, description: Option[String] = None, team: Option[String] = None, requiresAction: Option[Boolean] = None): Future[Seq[Vulnerability]] = {
+  def search(service: Option[String] = None, id: Option[String] = None, description: Option[String] = None, team: Option[String] = None): Future[Seq[Vulnerability]] = {
     val filters = Seq(
              service.map(s => Filters.equal("service", s)),
                   id.map(i => Filters.regex("id", i)),
          description.map(d => Filters.regex("description", d)),
-                team.map(t => Filters.equal("teams", t)),
-      requiresAction.map(r => Filters.equal("requiresAction", r))
+                team.map(t => Filters.equal("teams", t))
     ).flatten
 
     filters match {
       case Nil => collection.find().toFuture()
       case more => collection.find(Filters.and(more:_*)).toFuture()
```

Comment:
Rather than dupilcating the `collection.find` etc - you can just fold the filters - something like:
```scala
val filters = Seq(...).flatten.foldLeft(BsonDocument())(Filters.and)
```